src/components/Proposal/View/BiddingAndTenderingProcess.tsx

```diff
@@ -1,37 +1,124 @@
 import React, { useMemo } from 'react'
 
 import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import Time from 'decentraland-gatsby/dist/utils/date/Time'
 
-import { ProposalType } from '../../../entities/Proposal/types'
+import { ProposalAttributes, ProposalStatus, ProposalType } from '../../../entities/Proposal/types'
+import useProposal from '../../../hooks/useProposal'
 
-import ProposalProcess from './ProposalProcess'
+import ProposalProcess, { ProcessStatus } from './ProposalProcess'
 
 interface Props {
+  proposalId?: ProposalAttributes['id']
   proposalType: ProposalType
+  proposalStatus: ProposalStatus
+  proposalFinishAt: ProposalAttributes['finish_at']
+  linkedProposalId?: string
+  tenderProposalsTotal?: number
 }
 
-export default function BiddingAndTenderingProcess({ proposalType }: Props) {
+const getPitchConfig = (type: ProposalType, status: ProposalStatus) => {
+  if (type === ProposalType.Pitch && status === ProposalStatus.Active) {
+    return { status: ProcessStatus.Active, statusText: 'page.proposal_bidding_tendering.voting_ends' }
+  }
+
+  if (type === ProposalType.Pitch && status === ProposalStatus.Rejected) {
+    return { status: ProcessStatus.Rejected, statusText: 'page.proposal_bidding_tendering.initiative_rejected' }
+  }
+
+  if (type === ProposalType.Pitch && ProposalStatus.Passed) {
+    return { status: ProcessStatus.Passed, statusText: 'page.proposal_bidding_tendering.initiative_passed' }
+  }
+
+  if (type === ProposalType.Tender && ProposalStatus.Active) {
+    return { status: ProcessStatus.Passed, statusText: 'page.proposal_bidding_tendering.initiative_passed_title' }
+  }
+
+  return { status: ProcessStatus.Default, statusText: '' }
+}
+
+const getTenderConfig = (type: ProposalType, status: ProposalStatus, tenderProposalsTotal?: number) => {
+  if (type === ProposalType.Pitch && status === ProposalStatus.Passed) {
```

Comment:
It OK to check for pitch proposal type in the tender config?
Same goes for `getPitchConfig`, that checks for Tender type with Active status

Reply:
Tender statuses and texts depends on what kind of proposal you are viewing.